backend/framework/qlf/dashboard/bokeh/plots/plot2d/main.py

```diff
@@ -116,18 +116,42 @@ def circle(
             hover_color=hover_color,
             hover_alpha=hover_alpha,
             hover_line_color=hover_line_color,
-            radius=radius,
+            #radius=radius,
             fill_color=fill_color,
             hover_fill_color=hover_fill_color,
         )
         return self
 
+    def set_colorbar(self,z_value):
+        ''' Setup for colorbar
+        '''
+        dz = z_value
+
+        # removing NaN in ranges
+        dz_valid = [x if x > -999 else np.nan for x in dz]
+        dzmax, dzmin = np.nanmax(dz_valid), np.nanmin(dz_valid)
+
+        if np.log10(dzmax) > 4 or np.log10(dzmin) < -3:
+            ztext = ['{:4.2e}'.format(i) for i in dz_valid]
+            cbarformat = "%2.1e"
+        elif np.log10(dzmin) > 0:
+            ztext = ['{:5.2f}'.format(i) for i in dz_valid]
+            cbarformat = "%4.2f"
+        else:
+            ztext = ['{:6.2f}'.format(i) for i in dz_valid]
+            cbarformat = "%5.2f"
+        return ztext, cbarformat
+
     def colorbar(self, mapper, title='counts'):
+        formatter = PrintfTickFormatter(format="%4.2f")
+
         color_bar = ColorBar(color_mapper=mapper, label_standoff=16,
-                                title=title,
+                                title= "(Val-Ref)", #title,
```

Comment:
If this is going to be the default value for all colorbars please remove `colorbar_title`

Reply:
Sure, now I just changed the default value